### DISCUSSION TITLE
How do I move Directus to a Remote Server ( Docker Container )

### DISCUSSION THREAD

**Discussion by mudassar-islam** [directus/directus#2694] [Archive]
Can I just back-up the entire database, Re-install directus in a Docker Container and Restore the Database.

I'm not sure if it's as simple as that.

Is there a guide about migrating to another server properly.

Thank you :)

**Comment by benhaynes** [comment]
That should work, yes... but that doesn't touch the storage or assets.

**Comment by rijkvanzanten** [comment]
That's about it! Like @benhaynes mentioned, don't forget to copy over any uploads you have if you used the `local` file storage adapter before 🙂

**Comment by akki-akshat** [comment]
Hi @rijkvanzanten,

I want to move my current v9 Directus to another server, which all directories should I copy to the newer server?
PS. I am using one click installation from Digital Ocean, and I now want to move my content to another account of Digital Ocean altogether. I am using postgresql installed on the server as DB.

Thank you in advance for your help :)

**Reply by rijkvanzanten** [reply]
See https://github.com/directus/directus/discussions/5200
